vector/src/main/java/im/vector/VectorApp.java

```diff
@@ -393,6 +394,15 @@ private void stopActivityTransitionTimer() {
 
         mIsCallingInBackground = false;
         mIsInBackground = false;
+        setTheme();
+    }
+
+    private void setTheme() {
```

Comment:
I'd also suggest using a specific key (constant) to put in SharedPref instead of using a string that might be changed or deleted.